Taller152/src/paquetetres/EscribirArchivo.java

```diff
@@ -5,10 +5,21 @@
  */
 package paquetetres;
 
+import java.util.Formatter;
+
 /**
  *
  * @author reroes
  */
 public class EscribirArchivo {
-    
+      public static void mensajef(String datos) {
+        try {
+            Formatter out = new Formatter("data/Problema03");
+            out.format("%s\n", datos);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.